media-libs/opus/opus-1.3.recipe

```diff
@@ -24,28 +24,32 @@ Supported features are:
 * Dynamically adjustable bitrate, audio bandwidth, and frame size
 * Good loss robustness and packet loss concealment (PLC)
 * Floating point and fixed-point implementation"
-HOMEPAGE="http://www.opus-codec.org"
+HOMEPAGE="https://www.opus-codec.org/"
 COPYRIGHT="2001-2014 Xiph.org"
```

Comment:
Thanks for the update!
And sorry for asking an extra one, which I had missed... `sed s/2014/2018/`
```diff
-COPYRIGHT="2001-2014 Xiph.org"
+COPYRIGHT="2001-2018 Xiph.org"
```
(I promise I'll merge after this one :)
Thanks again!

Reply:
OK, it's there.